src/Makefile

```diff
@@ -42,10 +42,7 @@ CFLAGS?= $(STDFLAGS) $(ASAN)
 TST_CFLAGS:= -g $(STDFLAGS) #$(shell pkg-config --cflags check)
 GCOV_FLAGS?=-fprofile-arcs -ftest-coverage
 
-TST_LIBS?=-lcheck -lgmp
-ifeq ($(shell uname), Linux)
-	TST_LIBS=$(shell pkg-config --libs check) -lm -lrt -lpthread -lgmp
-endif
+TST_LIBS:=$(shell pkg-config --libs --cflags check) $(shell pkg-config --libs --cflags gmp)
```

Comment:
@bezlant Unfortunately, this makefile changes break build on both Mac & Apline. Please, see build log in GH Actions for this changes. Let's review it later.

```
/usr/lib/gcc/x86_64-alpine-linux-musl/10.3.1/../../../../x86_64-alpine-linux-musl/bin/ld: cannot find libasan_preinit.o: No such file or directory
/usr/lib/gcc/x86_64-alpine-linux-musl/10.3.1/../../../../x86_64-alpine-linux-musl/bin/ld: cannot find -lasan
collect2: error: ld returned 1 exit status
make: *** [Makefile:67: test] Error 1
Build test result: -1

4 warnings generated.
./math_ops/s21_div.c:[75](https://github.com/bezlant/int128_implementation/runs/6685925966?check_suite_focus=true#step:4:76):5: error: implicit declaration of function 'print_bits_r' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
    print_bits_r(temp);
    ^
1 error generated.
make: *** [objs/s21_div.o] Error 1
Build test result: -1
--------------------------

```